Frameworks/CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -707,11 +707,14 @@ CF_INLINE uintptr_t __CFISAForTypeID(CFTypeID typeID) {
 CF_INLINE bool __CF_IsBridgedObject(CFTypeRef obj) {
     CFRuntimeBase* object = (CFRuntimeBase*)obj;
     if (!object || 
-        (object->_cfisa == 0) || 
-        (object->_cfisa == (uintptr_t)(&_OBJC_CLASS__NSCFString)) || 
+        (object->_cfisa == 0)) {

```

Comment:
we are indicating that isa 0 is a CFObject in the __CF_IsCFObject and then we are covering the logic with a call into __CF_IsBridgedObject, for the __CF_IsCFObject, I assume nil is not a CFObject?